packages/inter-protocol/src/price/fluxAggregator.js

```diff
@@ -228,17 +228,18 @@ export const provideFluxAggregator = (
         seat.exit();
 
         return harden({
-          admin,
-
           invitationMakers,
+          oracle,
         });
       };
 
       return zcf.makeInvitation(offerHandler, INVITATION_MAKERS_DESC);
     },
     /** @param {string} oracleId */
-    deleteOracle: async oracleId => {
-      // FIXME how to GC and remove its powers?
+    removeOracle: async oracleId => {
+      trace('deleteOracle', oracleId);
+      const kit = oracles.get(oracleId);
+      kit.admin.disable();
```

Comment:
I wonder why `disable` rather than `revoke`. Can it be enabled again?

(thanks for adding the JS standard `revoke()` thing to https://github.com/dckc/awesome-ocap/wiki/CareTaker )

Reply:
that was the idea. right now there's no `enable()` method but the state is a boolean so it could be toggled.